fusion-documentation/src/main/minisite/content/fusion/examples.adoc

```diff
@@ -130,6 +130,9 @@ TIP: listening to `Start` event can enable a lazy instance (`@ApplicationScoped`
 
 TIP: an event can have more parameters, other parameters will be considered as injections (but the lookup will be destroyed after the method call if it is not `@ApplicationScoped`)
 
+TIP: When using the Launcher provided by the Fusion, you can use the shutdown hook to call the `container.close()` method and send the `Stop` event by settings the system properties
```

Comment:
Maybe mention "fusion configuration", ie system props, env var (useful for k8s).
Can also be worth mentioning signals it maps to (still k8s case)?